src/decorator-annotator.ts

```diff
@@ -279,7 +280,7 @@ class DecoratorRewriter extends Rewriter {
     super(sourceFile);
   }
 
-  process(): {output: string, diagnostics: ts.Diagnostic[]} {
+  process(): {output: string, diagnostics: ts.Diagnostic[], sourceMap: SourceMapGenerator} {
```

Comment:
I think that for the purpose of munging source maps together, returning a SourceMapGenerator is slightly better than returning a SourceMapConsumer.  I think we should keep the existing API.